package.json

```diff
@@ -63,6 +63,7 @@
     "@tailwindcss/postcss7-compat": "^2.0.2",
     "aewx-metar-parser": "~0.10.1",
     "byte-data": "^19.0.1",
+    "lodash": "^4.17.20",
```

Comment:
You really don't need lodash just to generate random values...

Reply:
It'll be added in my other PR anyway